predicates/traffic/segment.go

```diff
@@ -1,8 +1,9 @@
 package traffic
 
 import (
-	"math/rand"
+	"math/rand/v2"
```

Comment:
This is a complete unrelated change, please make sure you scope the change to what you want to achieve.
This change is a refactoring of math/rand to v2 and not redis cluster stupport.